spellfix/.gitignore

```diff
@@ -1,2 +1,3 @@
 /build
 .externalNativeBuild
+.cxx
```

Comment:
Whats this for?

Reply:
The new gradle plugin generates a new kind of native build artifact that we should ignore.

Reply:
Just like `.externalNativeBuild`, it is autogenerated when the project is imported and should not be checked into VCS.